docs/pages/database-access/guides/mysql-cloudsql.mdx

```diff
@@ -41,20 +67,18 @@ page and create another service account:
 
 ### Grant permissions
 
-Assign the Service Account the "Cloud SQL Admin" role:
-
-![Grant Cloud SQL Admin to Service Account](../../../img/database-access/guides/cloudsql/service-account-sql-admin-grant@2x.png)
+(!docs/pages/includes/database-access/cloudsql_grant_db_service_account.mdx!)
 
 <Admonition type="note" title="Service account permissions">
-  The default "Cloud SQL Admin" IAM role includes more permissions than the
-  Database Service needs to generate one-time user passwords. To further restrict
-  the service account, you can create a role that includes only the following
-  permissions:
+  "Service Account Token Creator", "Cloud SQL Viewer", and "Cloud SQL Admin"
+  IAM roles include more permissions than the Database Service needs. To further
+  restrict the service account, you can create a role that includes only the
+  following permissions:
   ```ini
-  # Used to download a list of database users.
-  cloudsql.users.list
```

Comment:
Was this permission only used for password auth, or was it just not necessary at all? I tried to find where we were using it before, but I couldn't find it.

Reply:
I couldn't find anywhere using it either. @r0mant do you recall where `cloudsql.users.list` is used?

Reply:
Looking at the guide history, it's been there since the very beginning but I can't remember why I put it there. If everything works without it, it's probably not needed.

Reply:
Sounds good. It does work without it when I was testing it.